services/testspace/testspace-test-summary.service.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
+
+const {
+  documentation,
+  testResultQueryParamSchema,
+  renderTestResultBadge,
+} = require('../test-results')
+const TestspaceBase = require('./testspace-base')
+
+module.exports = class TestspaceTests extends TestspaceBase {
+  static route = {
+    base: 'testspace/tests',
+    pattern: ':org/:project/:space+',
+    queryParamSchema: testResultQueryParamSchema,
+  }
+
+  static examples = [
+    {
+      title: 'Testspace tests',
```

Comment:
If I look at the preview dialog for the non-compact example, the compact option is checked.

Reply:
Good catch, copy/paste fail